src/ServiceInsight/Saga/SagaResources.xaml

```diff
@@ -25,4 +25,13 @@
     
     <SolidColorBrush x:Key="FailedMessageBackground" Color="#F9916F" />
     <SolidColorBrush x:Key="RetriedMessageBackground" Color="#F2F2F2" />
+
+    <Style x:Key="GrayNullTextStyle" TargetType="{x:Type TextBlock}">
```

Comment:
I prefer a functional name too. I guess we can always refactor later if we feel strongly enough.